requirements-base.txt

```diff
@@ -14,7 +14,7 @@ google-auth-oauthlib
 h11==0.9.0 # NOTE pinning due to uvicorn
 httpx
 jinja2
-jira
+jira==2.0.0
```

Comment:
I'm not sure we should pin the version. I'd rather update the plugin code when new versions are forcing us to do it.

Reply:
That makes sense. I can push undo the pinning. The rationale was changing when you decide to change rather than waiting for it to break and then change the version.